Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Pygmy flag update

Adding 10 CoAs for the Pygmy culture&implementing them.

# How to test:
Test if the CoAs have the desired dimensions.
Test if there are any issues with the CoAs.
Launch the game and check if the pygmies use the CoAs.
